src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,13 @@
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
```

Comment:
The FileWriterImpl class should not start with an empty line. Please remove the empty line before the class declaration.